automation/terraform/modules/testnet-alerts/templates/testnet-alert-rules.yml.tpl

```diff
@@ -735,26 +713,49 @@ groups:
       description: "{{ $value }} blocks have been produced that contains an invalid blockchain snark proof in last hour"
       runbook: "https://www.notion.so/minaprotocol/LowInvalidProofPerHour-b6e88b9ae84f47169e7f86017ab9e340"
 
-  - alert: LowPostgresBlockHeightGrowth
-    expr: min by (testnet) (increase(Coda_Archive_max_block_height {${berkeley_testnet}}  [${alert_timeframe}])) < 1
-    for: ${alert_evaluation_duration}
+  - alert: NodeRestarted
+    expr: count by (testnet) (Coda_Runtime_process_uptime_ms_total {${berkeley_testnet}} < 360000) > 0
     labels:
       testnet: "{{ $labels.testnet }}"
       severity: warning
     annotations:
-      summary: "{{ $labels.testnet }} rate of archival of network blocks in Postgres DB is lower than expected"
-      description: "The rate of {{ $value }} new blocks observed by archive postgres instances is low on network {{ $labels.testnet }}."
+      summary: "At least one of the nodes on {{ $labels.testnet }} restarted"
+      description: "{{ $value }} nodes on {{ $labels.testnet }} restarted"
+      runbook: "https://www.notion.so/minaprotocol/NodeRestarted-99a1cf710ff14aa6930a9f12ad5813a5"
+
+- Berkeley archive node alerts
```

Comment:
shouldn't we prefix this with `name:`
like:
`- name: Critical Alerts`

Reply:
Nice catch.